lib/build/bundle.js

```diff
@@ -108,7 +108,7 @@ exports.Bundle = class {
 
     if (loaderOptions.configTarget === this.config.name) {
       work = work.then(() => {
-        files.push({ contents: this.writeLoaderConfig(platform, loaderOptions) });
+        files.push({ contents: "_loadAureliaRequireJSConfig();"});
```

Comment:
Let's change the name of this function so that it doesn't mention RequireJS. In the future we'll support multiple module loaders. So maybe something like `_configureModuleLoader()`?